rootfs-builder/README.md

```diff
@@ -0,0 +1,62 @@
+# Building a rootfs for Kata Containers Guest OS #
+
+The Kata Containers guest OS is created using `rootfs.sh`.
+
+## Supported base OS ##
+
+The `rootfs.sh` script support different based OS `rootfs`. To build
+a `rootfs` based on a supported OS run:
+
+```
+$./rootfs.sh <distro>
+```
+
+To check the supported `rootfs` based OS run `$rootfs-builder/rootfs.sh
+-h`, it will show the supported values of `<distro>`
```

Comment:
- Missing period at end of sentence.
- I've run `./rootfs.sh -h` and don't see a list of supported distros though...?

Reply:
You should see an output like this, 
```bash
./rootfs-builder/rootfs.sh -h
USAGE: Build a Guest OS rootfs for Kata Containers image
rootfs.sh [options] <distro_name>

<distro_name> : Linux distribution to use as base OS.

Supported Linux distributions:

clearlinux
fedora
```
```

Reply:
Yep, sorry. I see it now! :smile: